org.openjkdf2.OpenJKDF2.yml

```diff
@@ -0,0 +1,32 @@
+app-id: org.openjkdf2.OpenJKDF2
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+
+command: openjkdf2-64
+
+modules:
+  - packaging/flatpak/glew/glew.json
+  - packaging/flatpak/glu/glu-9.json
```

Comment:
please use the shared-modules for these two: https://github.com/flathub/shared-modules/

Reply:
Ah, I'd run into a certificate error with the glu-9 download, but it seems to be resolved now? Or it's cached somewhere and still broken I guess.

Reply:
works fine everywhere else.